test_cases/test-assignfunction1.bs

```diff
@@ -0,0 +1,12 @@
+/*
+    Creates a function that adds two 
+*/
+
+int add(int a, int b) {
+    return a + b;
+}
+
+
+int addSeven (function add, int c) {
```

Comment:
shit we need to flesh out the syntax for creating a function that takes a function as a formal parameter

Reply:
so remove for now?

Reply:
not sure, but we need to flesh it out

Reply:
can write another test for this submission and flesh this out later

Reply:
ok